File: test-inject-app.js
// Test app for JSDoc injection
require('./packages/client-js/auto');

const { calculateTax, formatUser, filterItems } = require('./test-inject-lib');

const tax = calculateTax(100, 8.5);
console.log('Tax:', tax.tax);

const user = formatUser(
  { firstName: 'Alice', lastName: 'Smith', email: 'ALICE@EXAMPLE.COM' },
  'en-GB'
);
console.log('User:', user.display);

const filtered = filterItems([1, 2, 3, 4, 5], 3);
console.log('Filtered:', filtered.count);

console.log('Done!');

File: tests/test_inject_app.py
"""Test app for Python type hint injection."""
import trickle.auto

from test_inject_lib import calculate_tax, format_user, filter_items

tax = calculate_tax(100, 8.5)
print(f"Tax: {tax['tax']}")

user = format_user(
    {"first_name": "Alice", "last_name": "Smith", "email": "ALICE@EXAMPLE.COM"},
    "en-GB",
)
print(f"User: {user['display']}")

filtered = filter_items([1, 2, 3, 4, 5], 3)
print(f"Filtered: {filtered['count']}")

print("Done!")
